src/pybind/mgr/influx/module.py

```diff
@@ -132,7 +132,6 @@ def queue_worker(self):
                 start = time.time()
                 client = self.get_influx_client()
                 client.write_points(points, time_precision='ms')
-                client.close()
```

Comment:
@wido i just came across this change. and i think we should at least try to call this method to free the session. otherwise, the fd used by the `request.Session` will pile up before GC kicks in.

i am proposing an alternative fix at #28427. but i have not tested it.

Reply:
Good fix in #28427 ! I also thought about calling:

<pre>del client</pre>

Wouldn't that trigger GC immediately?

Reply:
@wido yes, it will help decrement the refcount, and i think, the refcount by then would be zero. so GC will be triggered. but i was wrong. GC does not close the `requests.Session` for us.

see https://github.com/kennethreitz/requests/blob/master/requests/sessions.py#L340 . so, `Session` does not have its `__del__(self)` method defined. put in other way, we'd better call `client.close()`.